tests/drivers/build_all/disk/app.overlay

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2024 TOKITA Hiroshi
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	test {
+		#address-cells = <1>;
```

Comment:
There are more disk devices we should add here- we can definitely build the ramdisk driver here, and I suspect we can also build the flash disk driver for native_sim, right?

Reply:
I got.
I will address this in another PR.